conans/test/unittests/tools/android/test_android_tools.py

```diff
@@ -0,0 +1,68 @@
+from conans.test.utils.mocks import ConanFileMock, MockSettings
+from conan.tools.android import android_abi
+
+
+def test_tools_android_abi():
+    settings_linux = MockSettings({"os": "Linux", "arch": "foo"})
+
+    for (arch, expected) in [
```

Comment:
This is usually better to be done with a `@parametrized` decorator in the function itself, so failures are easier to debug, but I'm ok with this either way :)

Reply:
I remember to have written tests with @pytest.mark.parametrize, then move to this implementation but I don't remember why. I can change back to parametrize.

Reply:
No big deal, if it works with parametrize easily, then do it, but don't waste time if you hit some issue.